lib/ansible/template/__init__.py

```diff
@@ -810,16 +810,16 @@ def _now_datetime(self, utc=False, fmt=None):
 
         return now
 
-    def _query_lookup(self, name, *args, **kwargs):
+    def _query_lookup(self, ansible_lookup_name, *args, **kwargs):
         ''' wrapper for lookup, force wantlist true'''
         kwargs['wantlist'] = True
-        return self._lookup(name, *args, **kwargs)
+        return self._lookup(ansible_lookup_name, *args, **kwargs)
 
-    def _lookup(self, name, *args, **kwargs):
-        instance = lookup_loader.get(name, loader=self._loader, templar=self)
+    def _lookup(self, ansible_lookup_name, *args, **kwargs):
```

Comment:
Use positional-only arguments instead:

```suggestion
    def _lookup(self, name, /, *args, **kwargs):
```